src/analysis/connectivity/score_inputs.sql

```diff
@@ -0,0 +1,2636 @@
+----------------------------------------
+-- INPUTS
+-- location: neighborhood
+----------------------------------------
+DROP TABLE IF EXISTS generated.neighborhood_score_inputs;
+
+CREATE TABLE generated.neighborhood_score_inputs (
+    id SERIAL PRIMARY KEY,
+    category TEXT,
+    score_name TEXT,
+    score NUMERIC(16,4),
+    notes TEXT,
+    human_explanation TEXT,
+    use_pop BOOLEAN,
+    use_emp BOOLEAN,
+    use_k12 BOOLEAN,
+    use_tech BOOLEAN,
+    use_univ BOOLEAN,
+    use_doctor BOOLEAN,
+    use_dentist BOOLEAN,
+    use_hospital BOOLEAN,
+    use_pharmacy BOOLEAN,
+    use_retail BOOLEAN,
+    use_grocery BOOLEAN,
+    use_social_svcs BOOLEAN,
+    use_parks BOOLEAN,
+    use_trails BOOLEAN,
+    use_comm_ctrs BOOLEAN,
+    use_transit BOOLEAN
+);
+
+
+-------------------------------------
+-- population
+-------------------------------------
+-- median pop access ratio
+INSERT INTO generated.neighborhood_score_inputs (
+    category, score_name, score, notes, human_explanation, use_pop
+)
+SELECT  'People',
+        'Median ratio of access to population',
+        quantile(pop_ratio,0.5),
+        regexp_replace('Ratio of population accessible by low stress
```

Comment:
Yes, great question. We're leaning towards maintaining the list of available scores, categories, and human-readable strings in the main database. This is a bit more flexible and extensible, and we should be able to (at least partially) guard against the metadata for each score getting out of sync with the list of scores being calculated by the analysis.

This would mean that all of the associated metadata in the intermediate table (old overall_scores table) could go away, and the new overall_scores table can just contain the score name key and value. 

It would be helpful if the score name set via the analysis was a short, unique, 'snake case' key that can double as the lookup key in the main database to retrieve the associated score metadata (category, label, long description, etc).  

If you're willing to make this change, we can communicate later this week on how to get the score metadata prepared.